ccm_web/server/src/canvas/canvas.interfaces.ts

```diff
@@ -40,6 +40,10 @@ export interface CanvasCourse extends CanvasCourseBase {
   course_code: string
 }
 
+export interface CanvasCourseSection extends CanvasCourseBase {
```

Comment:
Technically, `CanvasCourseBase` should be renamed `CanvasBaseObject`.  Then, most references to `CanvasCourseBase` should be replaced with `CanvasCourse`.

What do you think, @ssciolla?

Reply:
`Entity` is a pretty good choice.

I suggested this because it didn't make sense to me that `CanvasCourseSection` extends `CanvasCourseBase`, because section objects don't inherit from course objects.  Now that we're discussing it, I guess there's no reason that extension needs to be declared.  I can declare `CanvasCourseSection` as a new, independent type.

I guess if we want to pursue the type cleanup, it should be a new issue.